pkg/logproto/bloomgateway.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package logproto;
+
+import "gogoproto/gogo.proto";
+import "pkg/logproto/logproto.proto";
+
+option go_package = "github.com/grafana/loki/pkg/logproto";
+
+message FilterChunkRefRequest {
+  int64 from = 1 [
+    (gogoproto.customtype) = "github.com/prometheus/common/model.Time",
+    (gogoproto.nullable) = false
+  ];
+  int64 through = 2 [
+    (gogoproto.customtype) = "github.com/prometheus/common/model.Time",
+    (gogoproto.nullable) = false
+  ];
+  repeated logproto.ChunkRef refs = 3;
+  repeated logproto.LineFilterExpression filters = 4;
+}
+
+message ChunkIDsForStream {
```

Comment:
This can probably be represented more effectively by mirroring https://github.com/grafana/loki/pull/10780 which only encodes the fingerprint once per group. The chunk ID can be constructed from the `(tenant, fingerprint, start, end, checksum)`

```go
type Series struct {
	Fingerprint model.Fingerprint
	Chunks      []ChunkRef
}

type ChunkRef struct {
	Start, End model.Time
	Checksum   uint32
}
```